schemas/answer_actions/definitions.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$id": "https://eq.ons.gov.uk/answer_actions/definitions.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "redirect_to_list_add_block": {
+    "type": "object",
+    "description": "An action that redirects to the list add block",
+    "properties": {
+      "type": {
+        "enum": ["RedirectToListAddBlock"]
+      }
+    },
+    "additionalProperties": true,
+    "required": ["type"]
+  },
+  "remove_list_item_and_answers": {
+    "type": "object",
+    "description": "An action that redirects to the remove block",
+    "properties": {
+      "type": {
+        "enum": ["RemoveListItemAndAnswers"]
+      },
+      "params": {
+        "type": "object",
+        "properties": {
+          "block_id": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+          },
+          "list_name": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+          }
+        },
+        "additionalProperties": false,
+        "required": ["block_id", "list_name"]
+      }
```

Comment:
This isn't needed given we don't support this outside of a `PrimaryPersonListCollextor`?